_includes/settlement-reconciliation.md

```diff
@@ -175,6 +175,76 @@ The content of the files depends on the type of agreement you have made with Pay
+
+In short, it is a settlement feature where a company with a website or an app can attach specific prefix numbers to sub merchants selling their goods or services through the company. The prefix number is used to match the transactions and the merchant, so the settlement is automatically split between the sub merchants. If you run a site selling tickets to concerts, theatres, sporting events etc., each venue gets its own prefix number. If you run a funeral home, the sub merchants can be everything from flower delivery to charities.
+
+
+#### What we need from you as a company:
+
+* Send us a KYC form for each sub merchant you want to include. We will also do a KYC check on your sub merchants, providing extra security for you.
+* Give every sub merchant who sells goods/services at your website / in your app a unique prefix number. This needs to be included in the KYC form you send to us. We recommend using the same customer number they have in your system.
+* Attach the prefix number to all the items/goods/services the sub merchant sells through your website/app, so the goods/services can be matched to the correct merchant in our back office system.
+* A partner agreement is needed for the automatic deduction of revenue cuts and fees.
+
+
+#### How it works:
+
+1. We set up the sub merchant prefix in our systems.
+1. The prefix number is added in the subsite field in the API call when you create the payment for the goods/service.
+1. The customer selects payment method and completes the payment.
+1. The payment goes into the client funds account.
+1. Swedbank Pay matches the transaction and the merchant using the prefix number.
+1. The settlement is split and connected to the correct merchant.
+1. Revenue cuts for the super merchant and fees from Swedbank Pay are deducted automatically.
+1. Payout to the sub merchant is done.
+
+
+
+
+#### The upsides:
+
+* Since the sub merchants are connected to Swedbank Pay through the super merchant instead of having separate setups, this means that you:
+   * Only need one agreement for credit card, direct debit, Vipps, Swish, MobilePay,  invoice, credit account and payment gateway.
+   * Only need one acquiring agreement.
+   * Only need one Vipps/Swish certificate.
+   * Can add more payment methods easily, as it only has to be done once.
+   * Can set up new sub merchants quickly, as the only thing needed is a KYC form and a prefix number. This shortens the setup time for both you and us to a matter of hours.
+
+
+
+* The automatic settlement split and deduction of fees and revenue cuts minimizes the work for your accounting department, as you do not have to invoice your sub merchants.
+
+* The prefix split is available with all the payment methods we offer on our eCom platform:
```

Comment:
Is an extensive list needed if all instruments are supported? Also, we should retain a consistent language when speaking about our payment methods. I think "instruments" is used elsewhere.

```suggestion
* The prefix split is available with all the payment instruments we offer on our eCom platform:
```